sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/PreferencesBean.java

```diff
@@ -33,14 +34,15 @@
 public class PreferencesBean extends CourseDependentBean {
 
 	private static final long serialVersionUID = 1L;
+	private static final ResourceLoader rb = new ResourceLoader("pager");
 
 	public PreferencesBean() {
 		overviewSortColumn = "title";
 		overviewSortAscending = true;
 
 		rosterSortColumn = "studentName";
 		rosterSortAscending = true;
-		rosterMaxDisplayedRows = 50;
+		rosterMaxDisplayedRows = Integer.parseInt(rb.getString("pager_pageSize"));
```

Comment:
this makes no sense to me. why use a resource loader to get the pager size? are you assuming different locales want different pager sizes?